app/views/content/life-as-a-teacher/change-careers/how-to-change-careers-to-become-a-teacher/_article.html.erb

```diff
@@ -6,7 +6,8 @@
     <%= render Content::InsetTextComponent.new(
       header: "Non-UK citizens:",
       text: "<p>If you're not from the UK, find out about <a href=\"/non-uk-teachers/train-to-teach-in-england-as-an-international-student\">training to teach in England as a non-UK citizen</a>.</p>",
-      color: "purple"
+      color: "purple",
+      heading_tag: "h3"
```

Comment:
The heading tag looks right but the styling has now changed @spencerldixon ! Maybe one to discuss with @jenhadfield-dfe ?

![image](https://github.com/user-attachments/assets/2ab0b83c-d5ca-4773-92f1-be8190ea6ff5)